src/foam/nanos/crunch/ServerCrunchService.java

```diff
@@ -619,7 +620,7 @@ public boolean isRenewable_(X x, String capabilityId, boolean firstCall) {
     Capability capability = (Capability) capabilityDAO.find(capabilityId);
     UserCapabilityJunction ucj = crunchService.getJunction(x, capabilityId);
 
-    if ( ! capability.getEnabled() ) return false;
+    if ( capability.getLifecycleState() == LifecycleState.DELETED || capability.getLifecycleState() == LifecycleState.REJECTED ) return false;
```

Comment:
nothing to do with your pr - but makes me wonder why this is here. This should be on the capabilityDAO.

Reply:
Ya so looked - in the crunchService we have:
https://github.com/kgrgreer/foam3/blob/28d0cf196a50018c5c22def26327246a689a3f1e/src/foam/nanos/crunch/ServerCrunchService.java#L107
https://github.com/kgrgreer/foam3/blob/28d0cf196a50018c5c22def26327246a689a3f1e/src/foam/nanos/crunch/ServerCrunchService.java#L366
https://github.com/kgrgreer/foam3/blob/28d0cf196a50018c5c22def26327246a689a3f1e/src/foam/nanos/crunch/ServerCrunchService.java#L381
which are all just checking if capability null ... so this here deviates from other call expectations.
We will need to fix this sooner then later...can connect team do this?

Reply:
good catch will update